pkg/promlib/converter/testdata/prom-infos-frame.jsonc

```diff
@@ -11,9 +11,11 @@
 //      },
 //      "notices": [
 //          {
+//              "severity": "info",
```

Comment:
@zenador FYI - just letting you know that I'm updating to the latest plugin SDK changes which includes your [recent changes](https://github.com/grafana/grafana-plugin-sdk-go/pull/1185)